src/tsolvers/lasolver/Simplex.cpp

```diff
@@ -463,6 +468,9 @@ void Simplex::doGaussianElimination( )
 
 Delta Simplex::getValuation(LVRef v) const {
     Delta val(0);
+    if (tableau.isQuasiBasic(v)) {
+        (const_cast<Simplex*>(this))->quasiToBasic(v);
+    }
     if (isRemovedByGaussianElimination(v)) {
```

Comment:
Is this still a thing, can it happen?

Reply:
Gaussian elimination was removed in the next commit. It is not present in the final version, if that is what you were referring to.